docs/asciidoc/gettingStarted.adoc

```diff
@@ -44,33 +44,46 @@ case-by-case basis.
 ====
 
 [[getting-started-understanding-bom]]
-== Understanding the BOM
+== Understanding the BOM and versioning scheme
 
 Reactor 3 uses a BOM (Bill of Materials) model (since `reactor-core 3.0.4`, with the `Aluminium` release train).
 This curated list groups artifacts that are meant to work well together, providing
 the relevant versions despite potentially divergent versioning schemes in these artifacts.
 
-The BOM is itself versioned, using a release train scheme
-with a codename followed by a qualifier. The following list shows a few examples:
+Note the versioning scheme has changed between 3.3.x and 3.4.x (Dysprosium and Europium).
 
-[verse]
-Aluminium-RELEASE
-Californium-BUILD-SNAPSHOT
-Aluminium-SR1
-Bismuth-RELEASE
-Californium-SR32
+Artifacts follow a versioning scheme of `MAJOR.MINOR.PATCH-QUALIFIER` while the BOM is versioned using a CalVer inspired scheme of `YYYY.MINOR.PATCH-QUALIFIER`, where:
 
+ * `MAJOR` is the current generation of Reactor, where each new generation can bring fundamental changes to the structure of the project (which might imply a more significant migration effort)
+ * `YYYY` is the year of the first ga release in a given release cycle (like 3.4.0 for 3.4.x)
+ * `.MINOR` is a 0-based number incrementing with each new release cycle
+ ** in the case of projects, it generally reflects wider changes and can indicate a moderate migration effort
+ ** in the case of the BOM it allows discerning between release cycles in case two get first released the same year
+ * `.PATCH` is a 0-based number incrementing with each service release
+ * `-QUALIFIER` is a textual qualifier, which is omitted in the case of ga releases (see below)
+
+The first release cycle to follow that convention is thus `2020.0.x`, codename `Europium`.
+The scheme uses the following qualifiers (note the use of dash separator), in order:
+
+ * `-M1`..`-M9`: milestones (we don't expect more than 9 per service release)
+ * `-RC1`..`-RC9`: release candidates (we don't expect more than 9 per service release)
+ * `-SNAPSHOT`: snapshots
+ * _no qualifier_ for ga releases
+
+NOTE: snapshots appear high in the order above because, conceptually, they're always "the freshest pre-release" of any given PATCH.
```

Comment:
not 100% sure, but:
```suggestion
NOTE: snapshots appear higher in the order above because, conceptually, they're always "the freshest pre-release" of any given PATCH.
```